tests/integration/test_stampipes.py

```diff
@@ -3,8 +3,15 @@
 from pathlib import Path
 
 
-@pytest.mark.workflow(name='test_flag_qc_fail_improper_pair_and_nonnuclear_bam_reads')
+@pytest.mark.workflow(name="test_flag_qc_fail_improper_pair_and_nonnuclear_bam_reads")
```

Comment:
We have enough double quotes in our WDL. Can't let Black have these too.

Reply:
Also breaks the convention in all of our other py files.

Reply:
k.